projects/convert pdf to text/converter1.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+
+import PyPDF2
+
+
+BASEDIR = "C:\Documents\pdf_to_text\output.txt" # This is the sample base directory where all your text files will be stored if you do not give a specific path
```

Comment:
don't use any static file path
use temp directory in same folder because everyone do not have same path(Create if not exists)
```suggestion
BASEDIR = "pdf_to_text\output.txt"
```

Reply:
Ok

Reply:
Hey , I have added the temp directory